### ISSUE TITLE
No image at all

### ISSUE THREAD

**Issue by Derubosuke** [buchizo/ClaudiaIDE#204]
For some reason I cannot set a simple background, the first time I used the extension it worked fine, closed and opened the same project a few times and was fine, but after I opened a different project it stopped showing anything, from then it didn't work at all, already tried uninstalling and installing the extension, moving configs and cannot get a hint of what happened. I'm sorry if this is not the place to post this

**Comment by buchizo** [comment]
If you find .claudiaideconfig file in solution's folder, remove it. That file is specific settings for same folder's Visual Studio solution and priority high compare than general settings (in option menu).

**Comment by Derubosuke** [comment]
its not there, and actually, tried to create one, pressing save doesn't make one and creating one manually and saving doesn't put anything on it

**Comment by buchizo** [comment]
Hmm, If possible could you collect  Visual Studio log and share to here?

https://learn.microsoft.com/en-us/visualstudio/ide/reference/log-devenv-exe?view=vs-2019

**Comment by Derubosuke** [comment]
It has something to do with the color theme to use. Opening a project on light mode works totally fine, but on changing back to dark mode breaks again. If i still don't close the session and change back to light works again, but if I open visual studio on dark mode the extension doesn't work at all. Also trying to open visual a project on devenv with dark mode from the start crashes visual studio.
This is the one that opens visual studio on light mode and i changed to dark mode while on that session:
[ActivityLog1.txt](https://github.com/user-attachments/files/15508649/ActivityLog1.txt)
This is the one that crashes when trying to open on dark mode in devenv:
[ActivityLog2.txt](https://github.com/user-attachments/files/15508674/ActivityLog2.txt)
and the configs for both if they have any use:
![dark](https://github.com/buchizo/ClaudiaIDE/assets/30641042/88b0b171-08be-48a7-880a-8585bcd69920)
![light](https://github.com/buchizo/ClaudiaIDE/assets/30641042/5440da0d-400f-4e30-b34c-36c54b0f8df0)
Right now I'm ok, I'm just gonna leave it on light mode but if there is anything else i can do i can keep assisting on this

**Comment by buchizo** [comment]
What is `ClaudiaIDE - General` option value?

And I found crash log in your log. Maybe ClaudiaIDE crash by slideshow setting. However your settings is not use `slideshow`.

If possible,  could you set correct image folder path to `SlideShow` - `Directory Path`?

**Comment by buchizo** [comment]
Hmm, I guess your Visual Studio version is 17.0.34728.72 from log.
This is old version maybe, if possible could you upgrade your Visual Studio?

* the property description does not show in your option dialog screen shot. I guess this is wrong.. but it is Visual Studio feature..

if ClaudiaIDE `general` property is `use system`,  ClaudiaIDE is automatic choose `dark theme` or `light theme`  settings by system color theme. if possible you should use 'light' theme only.

**Comment by Derubosuke** [comment]
English is not my first language so its a bit hard to follow 😅 .
- Just updated to the last version and its still broken on dark mode.
- light mode still totally fine

**Comment by buchizo** [comment]
@Derubosuke 

I found root cause. please set `Dark theme` - `SlideShow` - `update interval` property to greater than or equal to `00:00:01`

![image](https://github.com/buchizo/ClaudiaIDE/assets/1254213/35973bef-8edc-4707-8396-b4528f0f4576)

**Comment by Derubosuke** [comment]
that is correct, it works
tysm
